server/app/repository/PersistedDurableJobRepository.java

```diff
@@ -0,0 +1,30 @@
+package repository;
+
+import com.google.common.collect.ImmutableList;
+import io.ebean.DB;
+import io.ebean.Database;
+import javax.inject.Inject;
+import models.PersistedDurableJob;
+
+/** Implements queries related to {@link PersistedDurableJob}. */
+public final class PersistedDurableJobRepository {
+
+  private final Database database;
+
+  @Inject
+  public PersistedDurableJobRepository() {
+    this.database = DB.getDefault();
+  }
+
+  public ImmutableList<PersistedDurableJob> listJobs() {
```

Comment:
listJobs -> getJobs.  list isn't a common idiom and it clearly returns a list from the definition.

Reply:
Done